docs/src/modules/components/ThemeViewer.tsx

```diff
@@ -189,13 +195,20 @@ export default function ThemeViewer({
 
   return (
     <TreeView
-      sx={{ bgcolor: '#333', color: '#fff', borderRadius: 1, p: 1 }}
       key={key}
       defaultCollapseIcon={<ExpandIcon />}
       defaultEndIcon={<div style={{ width: 24 }} />}
       defaultExpanded={defaultExpanded}
       defaultExpandIcon={<CollapseIcon />}
       {...other}
+      sx={{
+        color: '#FFF',
+        p: 1.5,
+        bgcolor: '#0F1924', // one-off code container color
+        borderRadius: 3,
+        border: '1px solid',
+        borderColor: '#1F262E', // primaryDark[700]
```

Comment:
If there is a blocker from using the theme token, I think the variable should be imported and use instead of hard-coded value.

e.g.
```js
import { blueDark } from '../path/to/brandingTheme'


bgcolor: blueDark[700]
```

Reply:
Sweet! For some reason, using the theme variables wasn't working on this file. Let me know if you know why! 😅 In the meantime, this is definitely better.